src/fragments/lib/in-app-messaging/js/customize.mdx

```diff
@@ -0,0 +1,109 @@
+## Integrate Custom Components
+
+Users may provide their own In-App Messaging UI components to override the default Amplify provided UI components by utilizing the `components` prop of `InAppMessagingProvider`.
+
+```js
+import {
+  InAppMessageDisplay,
+  InAppMessagingProvider
+} from 'aws-amplify-react-native';
+import { Home } from './src/Home';
+
+const MyBannerMessage = (props) => {
+  // ...Do something with props
+};
+
+const App = () => {
+  return (
+    <InAppMessagingProvider components={{ BannerMessage: MyBannerMessage }}>
+      <Home />
+      <InAppMessageDisplay />
+    </InAppMessagingProvider>
+  );
+};
+```
+
+Override component keys:
+
+- `BannerMessage` : override default message Banner UI component (top, middle, and bottom layouts)
+- `CarouselMessage` : override default message Carousel UI component
+- `FullScreenMessage` : override default message FullScreen UI component
+- `ModalMessage` : override default message Modal UI component
+
+<Callout informational>
+  For developer preview, only the BannerMessage and FullScreenMessage UI
+  components are provided by Amplify. CarouselMessage and ModalMessage
+  components must be manually provided to render Carousel and Modal layouts
+</Callout>
+
+## Integrate Custom Style
+
+Custom component styling can be applied to the default In-App Messaging UI components by passing the `style` prop to the `InAppMessagingProvider`, these styles will take precedence over default and In-App message payload style.
+
+```js
+import {
+  InAppMessageDisplay,
+  InAppMessagingProvider
+} from 'aws-amplify-react-native';
+import { Home } from './src/Home';
+
+const myInAppMessageStyle = {
+  BannerMessage: { container: { backgroundColor: 'lightgrey' } },
+  CarouselMessage: { body: { fontSize: 24 } }
+};
+
+const App = () => {
+  return (
+    <InAppMessagingProvider style={myInAppMessageStyle}>
+      <Home />
+      <InAppMessageDisplay />
+    </InAppMessagingProvider>
+  );
+};
+```
+
+The following optional UI component style properties can be per component:
+
+- `body` : text style applied to the message body
+- `closeIconButton` : view style applied to the close button
+- `closeIconColor` : string color value applied to close icon
+- `container` : view style to the primary container of the message component
+- `header` : text style applied to the message header
+- `image` : image style applied to the message image
+- `primaryButton` : view style applied to the primary message button
+- `secondaryButton` : view style applied to the secondary message button
```

Comment:
Let's remove the space between the closing back ticks and colons on these lines:

```suggestion
- `body`: text style applied to the message body
- `closeIconButton`: view style applied to the close button
- `closeIconColor`: string color value applied to close icon
- `container`: view style to the primary container of the message component
- `header`: text style applied to the message header
- `image`: image style applied to the message image
- `primaryButton`: view style applied to the primary message button
- `secondaryButton`: view style applied to the secondary message button
```